test/e2e_node/init_containers_test.go

```diff
@@ -54,13 +54,17 @@ type containerTestConfig struct {
 func (c containerTestConfig) Command() []string {
 	var cmd bytes.Buffer
 	// all outputs are in the format of:
-	// timestamp container-name message
-	fmt.Fprintf(&cmd, "echo `date +%%s` '%s Starting' >> /shared/output; ", c.Name)
-	fmt.Fprintf(&cmd, "echo `date +%%s` '%s Delaying %d' >> /shared/output; ", c.Name, c.Delay)
+	// time-since-boot timestamp container-name message
+
+	// The busybox time command doesn't support sub-second display. uptime displays in hundredths of a second, so we
+	// include both and use time since boot for relative ordering
+	timeCmd := "`date +%s` `cat /proc/uptime | awk '{print $1}'`"
+	fmt.Fprintf(&cmd, "echo %s '%s Starting' >> /dev/termination-log; ", timeCmd, c.Name)
+	fmt.Fprintf(&cmd, "echo %s '%s Delaying %d' >> /dev/termination-log; ", timeCmd, c.Name, c.Delay)
 	if c.Delay != 0 {
 		fmt.Fprintf(&cmd, "sleep %d; ", c.Delay)
 	}
-	fmt.Fprintf(&cmd, "echo `date +%%s` '%s Exiting' >> /shared/output; ", c.Name)
+	fmt.Fprintf(&cmd, "echo %s '%s Exiting' >> /dev/termination-log; ", timeCmd, c.Name)
```

Comment:
I'm not sure what implications might be for testing termination ordering. Should be covered I think. I think we can iterate - merging this to fix tests

Reply:
Sounds good, was trying to come up with anyway to get data out of the containers without writing to the filesystem.  I thought about looking at container logs, but this seemed to be a bit more straight forward.